ts/features/landingScreenMultiBanner/components/LandingScreenBannerPicker.tsx

```diff
@@ -0,0 +1,27 @@
+import * as React from "react";
+import { useIODispatch, useIOSelector } from "../../../store/hooks";
+import { landingScreenBannerToRenderSelector } from "../store/selectors";
+import { updateLandingScreenBannerVisibility } from "../store/actions";
+import { landingScreenBannerMap } from "../utils/landingScreenBannerMap";
+
+export const LandingScreenBannerPicker = () => {
+  const dispatch = useIODispatch();
+  const bannerToRender = useIOSelector(landingScreenBannerToRenderSelector);
+
+  const closeHandler = React.useCallback(() => {
+    if (bannerToRender) {
+      dispatch(
+        updateLandingScreenBannerVisibility({
+          id: bannerToRender,
+          enabled: false
+        })
+      );
+    }
+  }, [bannerToRender, dispatch]);
+
+  if (bannerToRender === undefined) {
+    return <></>;
```

Comment:
```suggestion
    return null;
```

I'd recommend returning `null` instead of a `Fragment`, as `null` is generally more appropriate when _nothing_ needs to be rendered.